### ISSUE TITLE
Dietpi Website Down

### ISSUE THREAD

**Issue by docgalaxyblock** [MichaIng/DietPi#2037]
Dietpi.com is reachable but access is Forbidden
That's all

**Comment by MichaIng** [comment]
@potter-91 
Jep sorry, might be caused by me, although error appeared after totally unrelated tasks I did on the server. I try to investigate. Seems that somehow server user permissions are not recognized. Even that webroot has correct www-data:www-data permissions, access fails with permission denied. Even trying sudo -u www-data:www-data /path/to/webroot/php_file fails with:
```
sudo: unable to stat /etc/sudoers: Permission denied
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin
```
But root and /etc folder permissions are correct, sudoers syntax and permissions correct, dpkg-reconfigure sudo made without resolving issue.

Even timesyncd fails (as it uses own user): `systemd-timesyncd[30255]: Failed to allocate manager: Permission denied`

So basically all actions by non-root users fail with permissions denied although folder/file permissions are correct. Seems a deeper system component that check/allows/denies permissions is broken.

APT <s>fails</s> shows warning as well:
`W: Download is performed unsandboxed as root as file '/var/cache/apt/archives/partial/xxxx.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)`
____________
`passwd[28650]: pam_unix(passwd:chauthtok): conversation failed`, but `pwck` manual check, permissions etc. on /etc/passws /etc/shadow etc. etc. are all correct.
Went through: https://www.cyberciti.biz/faq/linux-unix-bsd-osx-passwd-pam_chauthtok-error-in-service-module/
- Everything is as it should be.

**Comment by MichaIng** [comment]
Okay issue identified and solved. Accidentally chmod on root directory (not the subfolders) itself...
Silly mistake...

Many sorry to all affected users... Will mark as closed
